stacks/HANA-Scaleout-Standby/README.md

```diff
@@ -0,0 +1,189 @@
+# SAP HANA installation and configuration using Terraform and Ansible
+
+Terraform module and ansible roles to deploy the SAP HANA Scaleout stack. This stack deploys a master HANA node along with the required no.of worker nodes to form a HANA Scaleout cluster. All instances are deployed in the same zone. There is option to add additional worker nodes after the initial deployment. HANA Scaleout config is performed by a higher ansible role `ansible/roles/sap-hana-scaleout` that in turn calls lower ansible roles to do a specific task as part of the scaleout config.
+
+# Deployment Architecture
+
+### <img src="images/scaleout.png" width="500px">
+
+# Requirements
+
+* Terraform version `>=0.12`
+* Ansible version `>= 2.9.2`
+
+# Usage
+
+1. Terraform code for deploying the infrastructure required for installing and configuring SAP HANA Scaleout nodes is stored under `tf/`.
+
+2. Ansible roles for configuring HANA Scaleout on the GCE instances is stored under `sap-iac/ansible/roles`.
+
+3. Ansible playbook to deploy the HANA scaleout stack is `playbook.yml`.
+
+# Variables
+
+* All the ansible SAP HANA Scaleout configuration default values are defined in the higher level ansible role under `sap-iac/ansible/roles/sap-hana-scaleout/defaults/main.yml`.
```

Comment:
References to `sap-hana-scaleout` should be changed to `sap-hana-scaleout-standby`.